packages/vue-cli-plugin-vue-dash/generator/template/tests/e2e/specs/toolbar.spec.ts

```diff
@@ -0,0 +1,22 @@
+// https://docs.cypress.io/api/introduction/api.html
+describe('Toolbar page', () => {
```

Comment:
```suggestion
describe('Toolbar', () => {
```
Ce n'est pas une page !